Source/NSKeyValueObserving.m

```diff
@@ -1325,6 +1325,13 @@ - (void) removeObserver: (NSObject*)anObserver
   [iLock unlock];  
 }
 
+- (void) removeObserver: (NSObject*)anObserver
```

Comment:
Why do we have this method here? We don't define the method without context neither and we also don't have the corresponding add methods.

Reply:
It doesn't belong there...  I discovered the method should only exist where I have it implemented now, as a category on NSArray.